scripts/build/build_node/Platform/Linux/install-ubuntu-build-tools.sh

```diff
@@ -105,6 +71,10 @@ then
     then
         CMAKE_DISTRO_VERSION=3.20.1-0kitware1ubuntu20.04.1
         apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main'
+    elif [ "$UBUNTU_DISTRO" == "jammy" ]
+    then
+        # Ubuntu 22.04 already has an acceptable version of cmake
+        echo "Ubuntu 22.04's cmake package already at version 3.22.1"
```

Comment:
Do we still want to bump that up to 3.24 to support CMake presets?

Reply:
Since we don't use presets in the build scripts, its not particularly necessary. But we will have to if that changes

Reply:
Ok, I'll scope that into the work for adding presets